include/CMakeLists.txt

```diff
@@ -10,3 +10,18 @@ target_sources(VulkanLayerSettings PRIVATE
 	vulkan/layer/vk_layer_settings.hpp
 	vulkan/layer/vk_layer_settings_ext.h
 )
+
+
+add_library(VulkanLayerDispatchTable INTERFACE)
+add_library(Vulkan::LayerDispatchTable ALIAS VulkanLayerDispatchTable)
+
+target_sources(VulkanLayerDispatchTable PRIVATE
+    vulkan/layer/vk_layer_dispatch_table.h
+    vulkan/layer/vk_dispatch_table_helper.h
```

Comment:
Would it make sense to merge the 2 files? IE is there a point in having the files be separate?

Also why is it called `vk_layer_dispatch_table.h` why not just `vk_dispatch_table.h`?

Maybe I'm being dumb, but layer implies to me that these includes are only for layer developers.

Couldn't these files be used by the loader?

Reply:
We could combine them. I would hazard a guess that they are separate headers so that source files which only need the struct definition can get it, reducing compilation overhead. That said, with some of the cleaning up done here, significantly simpler than before, which makes some of the compilation overhead concern moot.

Reply:
I went ahead and combined them. Each file is only ~700 lines, which is not anything to worry about. And gets rid of the awkward "dispatch_table_helper" name to boot.

Reply:
Sounds good 👍🏾